satellite/console/consoleweb/server.go

```diff
@@ -192,6 +193,40 @@ func (server *Server) appHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// tokenRequestHandler authenticates User by credentials and returns auth token.
+func (server *Server) tokenHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	var err error
+	defer mon.Task()(&ctx)(&err)
+
+	var tokenRequest struct {
+		Email    string `json:"email"`
+		Password string `json:"password"`
+	}
```

Comment:
Might make sense to make this and the struct in L213 a global definition in the package, to have the ability to reuse it elsewhere in the future?

Reply:
We can define it globally when some other code will need it. For the nearest future I think it will be the only method which will use this structs.